### PR TITLE
Dtrade

### PR DESCRIPTION
**Pull request by CaptBoscho**

### CODE REVIEW

**Comment by yeyyosing** [comment]
Looks good to me

File: java/src/client/domestic/DomesticTradeController.java [hunk 1]
```diff
@@ -95,41 +130,120 @@ public void setAcceptOverlay(IAcceptTradeOverlay acceptOverlay) {
...
+					getTradeOverlay().setResourceAmountChangeEnabled(ResourceType.ORE, sendore < orecount, true);
+				}
 
 		}
+		if((sendbrick+sendwood+sendsheep+sendwheat+sendore > 0) && (receivebrick+receivewood+receiveore+receivesheep+receivewheat > 0) && tradePartner != -1){
+			getTradeOverlay().setTradeEnabled(true);
+		}
 	}
 
+	///TODO create trade object
```
Comment (dannyharding10) [inline java/src/client/domestic/DomesticTradeController.java]: Does this still need to be done

File: java/src/client/domestic/DomesticTradeController.java [hunk 2]
```diff
@@ -95,41 +130,120 @@ public void setAcceptOverlay(IAcceptTradeOverlay acceptOverlay) {
...
+				if(brickStatus){
+					receivebrick--;
+					getTradeOverlay().setResourceAmountChangeEnabled(ResourceType.BRICK, true, receivebrick > 0);
+				}else {
+					sendbrick--;
+					getTradeOverlay().setResourceAmountChangeEnabled(ResourceType.BRICK, sendbrick < brickcount, sendbrick > 0);
+				}
+				break;
+			case WOOD:
+				if(brickStatus){
```
Comment (emeraldarcher) [inline java/src/client/domestic/DomesticTradeController.java]: Should this be woodStatus?

**Comment by yeyyosing** [comment]
Corbin, danny is testing this pull request and it is super buggy.

**Pull request closed without merging**